packages/ui/src/components/va-split/VaSplit.vue

```diff
@@ -0,0 +1,349 @@
+<template>
+  <section
+    ref="splitPanelsContainer"
+    aria-label="split panels"
+    class="va-split"
+    :class="classComputed">
+    <div
+      class="va-split__panel"
+      :style="getPanelStyle('start')"
+    >
+      <slot name="start" />
+    </div>
+    <div class="va-split__dragger">
+      <div
+        class="va-split__dragger__overlay"
+        :style="draggerStyleComputed"
+        @mousedown.prevent="startDragging"
+        @touchstart.prevent="startDragging"
+        @dblclick.prevent="maximizePanel"
+        @contextmenu.prevent
+        @dragstart.prevent
+      >
+        <slot name="grabber">
+          <va-divider
+            class="va-split__dragger__default"
+            :vertical="!$props.vertical"
+          />
+        </slot>
+      </div>
+    </div>
+    <div
+      class="va-split__panel"
+      :style="getPanelStyle('end')"
+    >
+      <slot name="end" />
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType, ref, shallowRef, computed, watch, onMounted } from 'vue'
+import isString from 'lodash/isString.js'
+import isNumber from 'lodash/isNumber.js'
+import isUndefined from 'lodash/isUndefined.js'
```

Comment:
lol

Check this out: https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L12198

For me, using these lodash named methods makes belive that they're not simply checking for undefined, but they are. I would like to remove this.

Reply:
Yea, i know it. In previous implementation this check was used for many times, so i decided to use lodash instead of local check function. Now it's used in only one place and can be removed.